app/views/LocationShare/LiveLocationPreviewModal.tsx

```
@@ -0,0 +1,527 @@
+
+		if (trackerRef.current) {
+			const currentState = trackerRef.current.getCurrentState();
+			const idToStop = liveLocationId ?? (globalTrackerParams && globalTrackerParams.liveLocationId) ?? null;
+
+			trackerRef.current.stopTracking();
+			if (idToStop) {
+				try {
+					await markLiveLocationAsEnded(idToStop);
+				} catch (e) {
+					// best-effort; ignore but keep ESLint happy
+					void e;
```

ESLint

[error] Expected 'undefined' and instead saw 'void'.

(no-void)